content/zh/docs/reference/kubectl/_index.md

```diff
@@ -1,4 +1,1078 @@
+-->
+`kubectl delete` - 基于文件、标准输入或通过指定标签选择器、名称、资源选择器或资源来删除资源。
+
+<!--
+```shell
+# Delete a pod using the type and name specified in the pod.yaml file.
+kubectl delete -f pod.yaml
+
+# Delete all the pods and services that have the label '<label-key>=<label-value>'.
+kubectl delete pods,services -l <label-key>=<label-value>
+
+# Delete all pods, including uninitialized ones.
+kubectl delete pods --all
+```
+
+-->
+
+```shell
+# 使用 pod.yaml 文件中指定的类型和名称删除 Pod。
+kubectl delete -f pod.yaml
+
+# 删除所有带有 '<label-key>=<label-value>' 标签的 Pod 和服务。
+kubectl delete pods,services -l <label-key>=<label-value>
+
+# 删除所有 Pod，包括未初始化的 Pod。
+kubectl delete pods --all
+```
+
+<!--
+`kubectl exec` - Execute a command against a container in a pod.
+-->
+`kubectl exec` - 对 Pod 中的容器执行命令。
+
+<!--
+# Get output from running 'date' from pod <pod-name>. By default, output is from the first container.
+kubectl exec <pod-name> -- date
+
+# Get output from running 'date' in container <container-name> of pod <pod-name>.
+kubectl exec <pod-name> -c <container-name> -- date
+
+# Get an interactive TTY and run /bin/bash from pod <pod-name>. By default, output is from the first container.
+kubectl exec -ti <pod-name> -- /bin/bash
+-->
+
+```shell
+# 从 Pod <pod-name> 中获取运行 'date' 的输出。默认情况下，输出来自第一个容器。
+kubectl exec <pod-name> -- date
+
+# 运行输出 'date' 获取在 Pod <pod-name> 中容器 <container-name> 的输出。
+kubectl exec <pod-name> -c <container-name> -- date
+
+# 获取一个交互 TTY 并在 Pod  <pod-name> 中运行 /bin/bash。默认情况下，输出来自第一个容器。
+kubectl exec -ti <pod-name> -- /bin/bash
+```
+
+<!--
+`kubectl logs` - Print the logs for a container in a pod.
+-->
+`kubectl logs` - 打印 Pod 中容器的日志。
+
+<!--
+# Return a snapshot of the logs from pod <pod-name>.
+kubectl logs <pod-name>
+
+# Start streaming the logs from pod <pod-name>. This is similar to the 'tail -f' Linux command.
+kubectl logs -f <pod-name>
+-->
+
+```shell
+# 返回 Pod <pod-name> 的日志快照。
+kubectl logs <pod-name>
+
+# 从 Pod <pod-name> 开始流式传输日志。这类似于 'tail -f' Linux 命令。
+kubectl logs -f <pod-name>
+```
+
+<!--
+`kubectl diff` - View a diff of the proposed updates to a cluster.
+
+```shell
+# Diff resources included in "pod.json".
+kubectl diff -f pod.json
+
+# Diff file read from stdin.
+cat service.yaml | kubectl diff -f -
+```
+-->
+`kubectl diff` - 查看集群建议更新的差异。
+
+```shell
+# “pod.json”中包含的差异资源。
+kubectl diff -f pod.json
+
+# 从标准输入读取的差异文件。
+cat service.yaml | kubectl diff -f -
+```
+
+<!--
+## Examples: Creating and using plugins
+-->
+
+## 示例：创建和使用插件
+
+<!--
+Use the following set of examples to help you familiarize yourself with writing and using `kubectl` plugins:
+-->
+使用以下示例来帮助你熟悉编写和使用 `kubectl` 插件：
+
+<!--
+```shell
+# create a simple plugin in any language and name the resulting executable file
+# so that it begins with the prefix "kubectl-"
+cat ./kubectl-hello
+```
+```shell
+#!/bin/sh
+
+# this plugin prints the words "hello world"
+echo "hello world"
+```
+With a plugin written, let's make it executable:
```

Comment:
nit: Maybe it can be split to 2 or 3 part for next translator to get a better diff result.